class-tgm-plugin-activation.php

```diff
@@ -1757,7 +1760,7 @@ public function process_bulk_actions() {
 				unset( $plugin, $temp );
 
 				if ( is_wp_error( $api ) ) {
-					if ( WP_DEBUG === true ) {
+					if ( true === WP_DEBUG ) {
```

Comment:
Because I could have `define( 'WP_DEBUG', false )`.

WP already defines it as false if we've not defined it as something else, so the `WP_DEBUG` constant will always be defined - we just want to check it's been defined as true.